rclcpp/test/rclcpp/test_memory_strategy.cpp

```diff
@@ -362,12 +356,13 @@ TEST_F(TestMemoryStrategy, get_group_by_subscription) {
         memory_strategy()->get_group_by_subscription(subscription, weak_groups_to_nodes));
     }  // callback_group goes out of scope
     EXPECT_EQ(
-      callback_group,
+      nullptr,
       memory_strategy()->get_group_by_subscription(subscription, weak_groups_to_nodes));
   }  // Node goes out of scope
-  // NodeBase(SubscriptionBase->rcl_node_t->NodeBase) is still alive.
+  // NodeBase(SubscriptionBase->rcl_node_t->NodeBase) is still alive,
```

Comment:
The comment above says "node goes out of scope", why this comment says that node is still alive?

Reply:
This comment says the NodeBase is still alive, not Node, because the SubscriptionBase holds the rcl_node_t which holds the NodeBase.